.eslintrc

```diff
@@ -50,7 +51,8 @@
         "@typescript-eslint/ban-types": "off",
         "prefer-rest-params": "off",
         "prefer-spread": "off",
-        "@typescript-eslint/consistent-type-imports": [2]
+        "@typescript-eslint/consistent-type-imports": [2],
+        "@typescript-eslint/consistent-type-exports": [2]
```

Comment:
`@typescript-eslint` v5.70+ allows enforcing of "consistent-type-exports". I added it to help auto-fix type exports.